### ISSUE TITLE
Item size not known up front

### ISSUE THREAD

**Issue by anshckr** [kamilkp/angular-vs-repeat#112]
Is this implementation extensible for list items whose size is not known up front? I have a scenario where my item sizes are not known until the list is rendered.

**Comment by benjamincharity** [comment]
Yep: https://github.com/kamilkp/angular-vs-repeat#optional-parameters-attributes

> `vs-autoresize` - use this attribute without vs-size and without specifying element's size. The automatically computed element style will readjust upon window resize if the size is dependable on the viewport size
